src/node.h

```diff
@@ -337,9 +337,11 @@ NODE_EXTERN void FreeEnvironment(Environment* env);
 NODE_EXTERN Environment* GetCurrentEnvironment(v8::Local<v8::Context> context);
 
 // This returns the MultiIsolatePlatform used in the main thread of Node.js.
-// If NODE_USE_V8_PLATFORM haven't been defined when Node.js was built,
-// it returns nullptr.
+// If NODE_USE_V8_PLATFORM haven't been defined when Node.js was built, or
+// SetMainThreadMultiIsolatePlatform has not been called, it returns nullptr.
 NODE_EXTERN MultiIsolatePlatform* GetMainThreadMultiIsolatePlatform();
```

Comment:
This is an API that should also eventually go away, along with `per_process::v8_platform` imo… I’d consider being able to get the platform for a specified Environment much better.